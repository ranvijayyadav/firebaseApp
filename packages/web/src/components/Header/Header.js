import React from 'react';
import { UserConsumer } from './../../context/UserContext'
import { Navbar } from 'react-bootstrap';


const Header = (props) => {
    debugger;
    return (
        <UserConsumer>{({ loggedInUser }) =>
            <Navbar>
                <Navbar.Brand href="/">Firebase App {JSON.stringify(loggedInUser)}</Navbar.Brand>

            </Navbar>
        }
        </UserConsumer>
    );
};
export default Header;

