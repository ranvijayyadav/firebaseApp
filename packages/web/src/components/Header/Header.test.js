import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

let wrapped = shallow(<Header />);
describe('Header', () => {
    it("Should render Header properly", () => {
        expect(wrapped).toMatchSnapshot();
    });
})