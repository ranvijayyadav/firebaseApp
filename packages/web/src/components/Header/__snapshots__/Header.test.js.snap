// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render Header properly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
